pkg/controller/controller_test.go

```diff
@@ -2160,6 +2163,7 @@ func TestProvisionFromPVC(t *testing.T) {
 	testcases := map[string]struct {
 		volOpts              controller.ProvisionOptions
 		restoredVolSizeSmall bool
+		restoredVolSizeBig   bool
```

Comment:
We should probably add a description column to the test table so we don't have to infer it ourselves or guess what was intended.  I'll leave it up to you if you want to add that or I can submit a follow up PR to include it.  Just let me know.